FlipperKit.podspec

```diff
@@ -85,6 +85,7 @@ Pod::Spec.new do |spec|
                                "ONLY_ACTIVE_ARCH": "YES",
                                "DEFINES_MODULE" => "YES",
                                "HEADER_SEARCH_PATHS" => header_search_paths }
+    ss.xcconfig = { "GCC_PREPROCESSOR_DEFINITIONS" => "FB_SONARKIT_ENABLED=1", "OTHER_SWIFT_FLAGS" => "-Xcc -DFB_SONARKIT_ENABLED=1" }
```

Comment:
Sets the `FB_SONARKIT_ENABLED` macro for both (Obj)C and Swift targets.

Reply:
Note that I haven’t tested this in a Swift project, as I have none; this is entirely based on the previous docs about integrating with CocoaPods into a Swift project.

Reply:
We have a [SampleSwift](https://github.com/facebook/flipper/tree/master/iOS/SampleSwift) sample project too. Try to run it and make this changes over there too, if possible.

Reply:
Doh, I entirely missed that 🤦 Will do!

Reply:
Done 👍